### PR TITLE
✨Enable amp-bind for amp-google-document-embed source attr

### PR DESCRIPTION
**Pull request by cvializ**
Last feature before launch of `amp-google-document-embed` (#16907)

### CODE REVIEW

File: extensions/amp-google-document-embed/0.1/amp-google-document-embed.js [hunk 1]
```diff
@@ -99,6 +99,18 @@ export class AmpDriveViewer extends AMP.BaseElement {
     return this.loadPromise(iframe);
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    const attrs = ATTRIBUTES_TO_PROPAGATE.filter(
+        value => mutations[value] !== undefined);
+    this.propagateAttributes(
+        attrs, this.iframe_, /* opt_removeMissingAttrs */ true);
```
Comment (choumx) [inline extensions/amp-google-document-embed/0.1/amp-google-document-embed.js]: Is `[title]` is a valid binding?

Comment (cvializ) [reply extensions/amp-google-document-embed/0.1/amp-google-document-embed.js]: Ah it should be. Good catch

File: extensions/amp-google-document-embed/validator-amp-google-document-embed.protoascii [hunk 2]
```diff
@@ -37,6 +37,8 @@ tags: {  # <amp-google-document-embed>
     }
     blacklisted_value_regex: "__amp_source_origin"
   }
+  attrs: { name: "[src]" }
```
Comment (honeybadgerdontcare) [inline extensions/amp-google-document-embed/validator-amp-google-document-embed.protoascii]: nit: we usually have a comment before amp-bind attributes with `# amp-bind`. Not necessary though.

**Comment by cvializ** [comment]
Thanks for the reviews!

**Pull request merged**